starport/templates/ibc/oracle/static/x/{{moduleName}}/oracle.go.plush

```diff
@@ -0,0 +1,55 @@
+package <%= moduleName %>
+
+import (
+	"github.com/bandprotocol/bandchain-packet/obi"
+	"github.com/bandprotocol/bandchain-packet/packet"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	channeltypes "github.com/cosmos/cosmos-sdk/x/ibc/core/04-channel/types"
+	"<%= modulePath %>/x/<%= moduleName %>/types"
+)
+
+func (am AppModule) handleOraclePacket(
```

Comment:
We need to document this function `handleOraclePacket` and `handleOracleAcknowledgement` even though they are internal because their difference might not be trivial to the developer

`handleOraclePacket` handle the reception of the result from a previous query

`handleOracleAcknowledgement` handle the reception of the acknowledgment of a query before the result is received. It saves the id of this query

(Well this is how I understand them)